Mage.Sets/src/mage/cards/t/TurfWar.java

```diff
@@ -0,0 +1,220 @@
+        return new TurfWarCounterEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        MageObject sourceObject = game.getObject(source);
+        if (controller == null || sourceObject == null) {
+            return false;
+        }
+        boolean success = false;
+        for (Target target : source.getTargets()) {
+            for (UUID uuid : target.getTargets()) {
+                Permanent permanent = game.getPermanent(uuid);
+                if (permanent != null && permanent.addCounters(CounterType.CONTESTED.createInstance(), source, game)) {
+                    game.informPlayers(sourceObject.getLogName() + ": " + controller.getLogName()
+                            + " puts a contested counter on " + permanent.getLogName());
+                    success = true;
+                }
+            }
+        }
+        return success;
+    }
+}
+
+enum TurfWarAdjuster implements TargetAdjuster {
+    instance;
+
+    @Override
+    public void adjustTargets(Ability ability, Game game) {
+        ability.getTargets().clear();
+        for (UUID playerId : game.getState().getPlayersInRange(ability.getControllerId(), game)) {
+            Player player = game.getPlayer(playerId);
+            if (player == null) {
+                continue;
+            }
+            FilterLandPermanent filter = new FilterLandPermanent("land controlled by " + player.getName());
+            filter.add(new ControllerIdPredicate(playerId));
+            ability.addTarget(new TargetLandPermanent(filter));
+        }
+    }
+}
+
+class TurfWarTriggeredAbility extends TriggeredAbilityImpl {
+
+    public TurfWarTriggeredAbility() {
+        super(Zone.BATTLEFIELD, new TurfWarControlEffect());
+    }
+
+    private TurfWarTriggeredAbility(final TurfWarTriggeredAbility ability) {
+        super(ability);
+    }
+
+    @Override
+    public TurfWarTriggeredAbility copy() {
+        return new TurfWarTriggeredAbility(this);
+    }
+
+    @Override
+    public boolean checkEventType(GameEvent event, Game game) {
+        return event.getType() == GameEvent.EventType.DAMAGED_PLAYER;
+    }
+
+    @Override
+    public boolean checkTrigger(GameEvent event, Game game) {
+        DamagedPlayerEvent damageEvent = (DamagedPlayerEvent) event;
+        UUID creatureId = damageEvent.getSourceId();
+        UUID playerId = damageEvent.getPlayerId();
+        if (damageEvent.isCombatDamage()) {
+            for (Permanent permanent : game.getBattlefield().getAllActivePermanents(StaticFilters.FILTER_LAND, playerId, game)) {
+                if (permanent.getCounters(game).getCount(CounterType.CONTESTED) > 0) {
+                    this.getEffects().setValue("creature", creatureId);
+                    this.getEffects().setValue("player", playerId);
+                    return true;
+                }
+            }
+        }
+        return false;
```

Comment:
This and other checks need to make sure that both controller of the attacking creature and the player taking damage are within the range of influence of the Turf War's controller. (see #9269).

Reply:
Added range of influence check.